### PR TITLE
Expand prose about preventing compatibility mouse events and make it normative

### LINKED ISSUE #369: Add note explicitly about mouse compat events, preventDefault, and passive events
it's implicit at the moment, but would be good to just make explicit

https://github.com/w3c/pointerevents/issues/360#issuecomment-824407307

### PR DESCRIPTION
**Pull request by patrickhlauke**
* split the awkward "Also," into a separate paragraph (still inside the note)
* add an additional paragraph in the note about passive event listeners (which ignore `preventDefault` / where the default action can't be prevented) - see https://github.com/w3c/pointerevents/issues/360#issuecomment-824407307
* make the note itself normative / remove the "Note" around it

Closes https://github.com/w3c/pointerevents/issues/369

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/pull/403.html" title="Last updated on Aug 17, 2021, 11:22 PM UTC (7866b69)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/403/cbd05e0...7866b69.html" title="Last updated on Aug 17, 2021, 11:22 PM UTC (7866b69)">Diff</a>

### CODE REVIEW

**Review by smaug---- — CHANGES_REQUESTED** [review]
Calling preventDefault() from a passive event listener is defined already in the DOM spec so I think it is just a bit confusing to explain it here too 
https://dom.spec.whatwg.org/#set-the-canceled-flag

**Comment by patrickhlauke** [comment]
Still wondering if there's a value in mentioning it - it caught me by surprise when it was mentioned, but then again I'm not hip with the DOM spec. Maybe rewording it slightly but keeping it as a nod/reminder? To make clear that compat events (when generated) are part of the default action, and that of course that default action won't be preventable in those situations? But yeah, not precious about this...if it's clear enough/obvious for others, happy to remove that para (but keep the slight tweak in structure for the note)

**Comment by smaug----** [comment]
Perhaps some note could work here?

**Review by mustaqahmed — APPROVED** [review]

**Comment by patrickhlauke** [comment]
As discussed in today's call https://www.w3.org/2021/08/04-pointerevents-minutes.html added an explicit `[[DOM]]` reference there to make it clearer that the concepts are defined in DOM

**Comment by patrickhlauke** [comment]
However, with our newfound interest in normative/non-normative ... wondering if that second note for https://pr-preview.s3.amazonaws.com/w3c/pointerevents/pull/403.html#compatibility-mapping-with-mouse-events should actually just be normal normative prose rather than a non-normative note. Thoughts?

**Review by flackr — APPROVED** [review]

**Comment by mustaqahmed** [comment]
We have [pointerevent_mouse-pointer-preventdefault.html](https://wpt.fyi/results/pointerevents/compat/pointerevent_mouse-pointer-preventdefault.html?label=master&label=experimental&aligned&view=subtest) to cover almost all of this PR.  However, we still need a WPT for compat events for passive PointerEvent listeners.

**Comment by smaug----** [comment]
https://github.com/web-platform-tests/wpt/pull/38525

**Comment by mustaqahmed** [comment]
While working on a test on passive event listeners, I realized that this behavior is not normative!  It seems to me that the "note" we added here could be normative...thoughts?

**Comment by mustaqahmed** [comment]
> While working on a test on passive event listeners, I realized that this behavior is not normative! It seems to me that the "note" we added here could be normative...thoughts?

In today's PEWG meeting we agreed to make this normative.

**Comment by patrickhlauke** [comment]
> In today's PEWG meeting we agreed to make this normative.

done https://github.com/w3c/pointerevents/pull/465

**Comment by mustaqahmed** [comment]
The WPT landed through https://github.com/web-platform-tests/wpt/pull/38895
